oauth2_http/java/com/google/auth/oauth2/CertificateIdentityPoolSubjectTokenSupplier.java

```diff
@@ -50,6 +50,7 @@
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import jdk.internal.joptsimple.internal.Strings;
```

Comment:
You probably meant Guava's Strings from com.google.common.base

Reply:
Thanks for catching this!